testng-core/src/test/java/test/junitreports/TestClassContainerForGithubIssue1265.java

```diff
@@ -10,10 +10,10 @@ public class TestClassContainerForGithubIssue1265 {
   public abstract static class ParentTest {
 
     @BeforeSuite
-    public synchronized void startEverything() throws Exception {}
+    public void startEverything() {}
 
     @AfterSuite
-    public synchronized void shutdownContainer() throws Exception {}
```

Comment:
It is no longer the same test case. It it expected?

Reply:
Based on my analysis, the `synchronized` keyword has got nothing to do with https://github.com/testng-team/testng/issues/1265 So I removed it.